### ISSUE TITLE
[feature request] worked with ace-window.

### ISSUE THREAD

**Issue by zw963** [roman/golden-ratio.el#68]
[ace-window](https://github.com/abo-abo/ace-window) is a easy way switch multi-window/frame quickly.

![picture](http://oremacs.com/download/ace-window.gif)

Thanks.

**Comment by ChillarAnand** [comment]
Is it working with ace-window?

I've installed both packages. When i switch window using ace-window, golden ratio is not resizing.

**Comment by FrancisMurillo** [comment]
I have a similar problem with `window-numbering-mode` where when I switched windows it would not resize accordingly. The approach might be to advice `select-window` or to create hooks in either `ace-window` or `golden-ratio` or what have you. The answer to **advising `select-window`**(#57 ) might be good enough.

Without being too wordy on the reference, this snippet might help you friend... I think.
```emacs-lisp
;; Refer to issue #57 for the complete code, this is just for ease
(add-hook 'buffer-list-update-hook #'golden-ratio) 
(add-hook 'focus-in-hook           #'golden-ratio)
(add-hook 'focus-out-hook          #'golden-ratio)
```

**Comment by therockmandolinist** [comment]
I was able to get this working with `ace-window` using the following advice (new advice syntax):
```emacs-lisp
(define-advice select-window (:after (window &optional no-record) golden-ratio-resize-window)
    (golden-ratio)
    nil)
```
